app/connector/support/processor/src/main/java/io/syndesis/connector/support/processor/ErrorMapper.java

```diff
@@ -13,18 +13,20 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.syndesis.connector.apiprovider;
+package io.syndesis.connector.support.processor;
 
 import java.io.IOException;
 import java.util.Collections;
 import java.util.Map;
 
 import io.syndesis.common.util.json.JsonUtils;
+
 import org.apache.camel.util.ObjectHelper;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.fasterxml.jackson.core.type.TypeReference;
+import com.google.common.base.CaseFormat;
```

Comment:
Perhaps we should not introduce a Guava dependency in our own code, that's a really heavy dependency with many conflicting/incompatible versions used in libraries we might pull in so using it could lead to runtime errors.

Reply:
It's already in our dependency tree, so why not use it?

Reply:
Even ban it via enforcer, though that might be a bit harsh (I did ban hamcrest earlier). We brought it in via swagger.io (I think) and swagger.io was within syndesis-common, now we no longer depend on swagger.io so let's not introduce new dependencies on Guava. We can refactor the existing ones as we go.

Reply:
Removing Guava as direct dependency so we don't propagate it https://github.com/syndesisio/syndesis/pull/7380